src/models/src/NITableViewModel.h

```diff
@@ -119,10 +119,10 @@ typedef enum {
  */
 @interface NITableViewModelFooter : NSObject
 
-+ (id)footerWithTitle:(NSString *)title;
-- (id)initWithTitle:(NSString *)title;
++ (nonnull id)footerWithTitle:(nonnull NSString *)title;
+- (nonnull id)initWithTitle:(nonnull NSString *)title;
```

Comment:
I suppose since we made the title property nullable we could technically make the title parameters on these functions nullable as well... Or we could just leave them as nonnull because we don't want to encourage folks to pass in null values.

Reply:
May leave as nonnull and encourage folks to use alloc/init instead for the nil case. I wager longer-term we might get rid of the default `init` and mark `initWithTitle` as a designated initializer.